ui/app/selectors/selectors.js

```diff
@@ -455,3 +456,29 @@ export function getSwapsEthToken(state) {
     }),
   };
 }
+
+export function getShowWhatsNewPopup(state) {
+  return state.appState.showWhatsNewPopup;
+}
+
+function getNotificationFilters(state) {
+  const currentNetworkIsMainnet = getIsMainnet(state);
+  const swapsIsEnabled = getSwapsFeatureLiveness(state);
+
+  return {
+    1: !currentNetworkIsMainnet || !swapsIsEnabled,
```

Comment:
This seems backwards from what I'd expect :thinking: e.g. `Array.prototype.filter` returns `true` for things that we want, but this returns `false` for things that we want.

On a similar note, it would be great to have JSDoc comments for these exported functions

Reply:
Also, should these filters be defined in `shared/notifications/index.js` with the rest of the notification definition? It seems like a part of the definition to me.

Reply:
> Also, should these filters be defined in shared/notifications/index.js with the rest of the notification definition? It seems like a part of the definition to me.

I think not, because of 
![Screenshot from 2021-03-19 12-07-57](https://user-images.githubusercontent.com/7499938/111797109-cae93b80-88ab-11eb-9305-482162d975b6.png)

Reply:
Ah, right! The comment in that screenshot was suggesting that the action functions be migrated to the UI, but it appears that you've left them in the shared module. I'm guessing you left them there because they didn't directly depend upon any UI modules, but this is completely incidental - future action functions might, and likely will. They are still only used in the UI. The only thing needed by the background controller is map of notification IDs to dates.

Reply:
Okay, I've moved that out of shared in 9462cd22e

Reply:
Have you given any thought to the first two suggestions in this thread?

Reply:
> This seems backwards from what I'd expect thinking e.g. Array.prototype.filter returns true for things that we want, but this returns false for things that we want.

Given that we are no longer having a notification on swaps in the extension (see this thread), there is no longer a need for this filtering. It has been removed as part of 9dbdb76, which removes all code required for the notification that would bring the user to the swaps route.

Reply:
> On a similar note, it would be great to have JSDoc comments for these exported functions

the jsdoc comments for `getSortedNotificationsToShow` have been added